app/javascript/src/components/Dashboard/Notes/Pane/constants.js

```diff
@@ -0,0 +1,12 @@
+const INITIAL_CONTACT_VALUES = ["Oliver Smith", "Sam Smith"];
+
+const INITIAL_TAG_VALUES = [
+  "Getting Started",
+  "Onboarding",
+  "User Flow",
+  "UX",
+  "Bugs",
+  "V2",
+];
+
+export { INITIAL_CONTACT_VALUES, INITIAL_TAG_VALUES };
```

Comment:
We don't need to create a folder called `Pane`. 
We could name the folder as `NewContact` and inside of it we have `index.jsx` that acts as the entry point. If you want, you could extract out the logic to `Form.jsx`. But generally speaking, let's not use `Pane, Form, Modal` as folder or file names.